### ISSUE TITLE
cURL error 60: SSL certificate problem: unable to get local issuer certificate 

### ISSUE THREAD

**Issue by nmajmari** [googleapis/google-api-php-client#1944]
#### Environment details

  - OS:Windows 10 Pro
  - PHP version: 7.3.19
  - Package name and version: google/apiclient:"^2.7.1"

#### Steps to reproduce
I am trying to validate Google Id Token in my Guard Authenticator ( Symfony 4.4). I get the id token from a get request. When i try to validate the token using code below i get this error: 
cURL error 60: SSL certificate problem: unable to get local issuer certificate (see https://curl.haxx.se/libcurl/c/libcurl-errors.html)
It is wired because it has worked before. 
#### Code example
```php
  $client = new \Google_Client(['client_id'=>null]);
 $payload = $client->verifyIdToken($credentials['id_token']);
```

Thanks!

**Comment by jdpedrie** [comment]
This error indicates a misconfiguration on the machine running your application. Take a look at the various solutions discussed [here](https://github.com/googleapis/google-api-php-client/issues/843) and let me know if that helps.

**Comment by nmajmari** [comment]
Thanks mate i fixed it. I followed instructions from this medium article [here](https://medium.com/@narendravaghela/how-to-fix-curl-error-60-ssl-certificate-problem-80e7dafafa57) mentioned there. 
Thank you very much!
